EXAMPLES.md

```diff
@@ -50,6 +50,8 @@ using Shopify.Unity;
 using Shopify.Unity.SDK;
 
 void Start () {
+    const int productsPerPage = 10;
```

Comment:
I feel like it's better just to tell people to use the default value since it gives us control to be able to have a "safe" value. Also the example is simpler without this.

Reply:
Updated!